test/unit/vl-mapactions-map-with-actions.test.js

```diff
@@ -31,7 +31,7 @@ describe('map with actions', () => {
     setTimeout(() => {
       callback();
       done();
-    }, VlMapWithActions.CLICK_COUNT_TIMEOUT);
+    }, VlMapWithActions.CLICK_COUNT_TIMEOUT + 10);
```

Comment:
random? :)

Reply:
Moet ik er + 1 van maken dan? In de source code zit een setTimeout met die waarde en dat was net niet OK om ook in de test zo lang te wachten ...

Reply:
Het is gewoon onduidelijk waarom die +1 of +10 of +x nodig is.

Reply:
+1 van gemaakt en comment toegevoegd